src/pocketmine/Player.php

```diff
@@ -2310,7 +2321,7 @@ public function handleDataPacket(DataPacket $packet){
 				break;
 
 			case ProtocolInfo::REMOVE_BLOCK_PACKET:
-				if($this->spawned === false or !$this->isAlive()){
+				if($this->spawned === false or !$this->isAlive()()){
```

Comment:
Yep.  I thought that those weren't mine but it turns out they were.  I searched and replaced the method without parenthesis and so it left them